pkg/session_recording/recordings.jsx

```diff
@@ -468,12 +467,7 @@
             let options = {follow: true, count: "all", since: this.state.dateSince, until: this.state.dateUntil};
 
             if (this.state.recordingID !== null) {
-                let parts = this.state.recordingID.split('-', 3);
-                matches = matches.concat([
-                            "_BOOT_ID=" + parts[0],
-                            "TLOG_SESSION=" + parts[1],
-                            "_PID=" + parts[2]
-                ]);
+                matches = "TLOG_REC=" + this.state.recordingID;
```

Comment:
This overrides the _UID= condition and makes `matches` a string, not an array. We need to have the _UID condition for authenticating genuine tlog-rec messages.